addons/calendar/models/calendar_event.py

```diff
@@ -556,6 +563,29 @@ def action_sendmail(self):
                 meeting.attendee_ids._send_mail_to_attendees('calendar.calendar_template_meeting_invitation')
         return True
 
+    def action_mass_mailing(self):
+        partners_ids = self.mapped('partner_ids')
+        if not partners_ids:
+            raise UserError(_("There are no attendees on these events"))
+        compose_form = self.env.ref('mail.email_compose_message_wizard_form', False)
+        default_partners = partners_ids and partners_ids.ids
+        compose_ctx = dict(
+            default_use_template=False,
+            default_composition_mode='mass_mail',
+            default_partner_ids=default_partners,
```

Comment:
You should not set partner_ids but instead give a model and a pair of ids to mail.